aws-redshift-clustersubnetgroup/src/test/java/software/amazon/redshift/clustersubnetgroup/TestUtils.java

```diff
@@ -67,7 +67,6 @@ public class TestUtils {
             .description(DESCRIPTION)
             .clusterSubnetGroupName(SUBNET_GROUP_NAME)
             .subnetIds(SUBNET_IDS)
-            .tags(TAGS)
             .build();
 
     final static DescribeTagsResponse DESCRIBE_TAGS_RESPONSE = DescribeTagsResponse.builder()
```

Comment:
Can you confirm if we want to remove Tags, since we use the Tagris service to add/update/remove tags ?

Reply:
our CFN handler can still handle Tagging feature. only we just don't use it to do drifct detection